src/tests/test-helpers.js

```diff
@@ -114,3 +114,28 @@ export function createAdminUser(slug) {
 
   return user;
 }
+
+export function createProject(
+  projectName = randomUUID(),
+  usernameOrUser = randomUUID(),
+) {
+  let user;
+
+  if (typeof usernameOrUser === "string") {
+    user = createUser(usernameOrUser);
+  } else {
+    user = usernameOrUser;
+  }
+
+  // The user must be enabled for various Project calls to succeed
+  User.enableUser(user);
+
+  const project = Project.createProjectForUser(user, projectName);
+  return project;
+}
+
+export function createStarterProject(projectName, usernameOrUser) {
+  const project = createProject(projectName, usernameOrUser);
+  Models.StarterProject.create({ project_id: project.id });
```

Comment:
This will require updating `clearTestData` to also clear the starters table. Looking at what we're doing, and probably the admin flag table as we're not clearing that right now either.

Reply:
ahhh good call, thanks!